### ISSUE TITLE
How can I build this?

### ISSUE THREAD

**Issue by brianpipa** [rhollister/goodreads#37]
I'm a java dev by trade... pulled this code, tried a few things to build/package it but nothing worked. Any quick tips to get it built?

**Comment by rhollister** [comment]
Sure! Which browser are you trying to build it for? Do you have access to a bash shell (OSX, Linux) or cygwin (Windows)?

From a bash command line, in the `build/` directory run: `./build_chrome.sh`. And it should create a zip file in the parent directory that you can then load as a developer extension in Chrome. You might have to first run `chmod 755 *.sh` in `build/` if it gives you permission errors about running it.

**Comment by brianpipa** [comment]
Hmmm...
ok... I was shooting for Chrome. I use linux as my desktop. I tried that,
did have to chmod +x the script, but when I run it nothing appears to
happen.
AHA
when I firt run it I get this:
~/workspace/goodreads/build$ ./build_chrome.sh
rm: cannot remove 'availablereads-chrome.zip': No such file or directory

that error threw me. I got it now - thanks! If it were me, I'd have that
build script output some progress/status as it goes :0

But I have it building now. Thanks so much. I have been using this for ages
and it works great.
---
Brian Pipa
